snuba/datasets/cdc/groupassignee.py

```diff
@@ -12,4 +13,8 @@ class GroupAssigneeDataset(Dataset):
     """
 
     def __init__(self) -> None:
-        super().__init__(default_entity=EntityKey.GROUPASSIGNEE)
+        super().__init__(
+            all_entities=[
+                get_entity(EntityKey.GROUPASSIGNEE),
```

Comment:
nit: No need to do the get entity call in every dataset, let's just pass the entity keys and do that in base as well?